### PR TITLE
Updates README and adds docs

### PR DESCRIPTION
**Pull request by infiniterasadev**
- Update the README to be a little more welcoming.
- Create a /docs and /images folder to contain documentation and screenshots
- Added setup.md which is a first run at an updated setup guide.
- Added screenshot.jpg which is a picture of the servers running for some flare on the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # Rasa.NET
-A C# version of our amazing game!
+A C# implementation of a game and authentication server for Tabula Rasa running on .NET Core.
```
Comment (Blumster) [inline README.md]: Don't include copyrighted names for legal reasons.

File: docs/setup.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# Setup Guide
+This guide will help you install and setup the required tools to run Tabula Rasa using Rasa.NET. There are a few steps:
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons.

File: docs/setup.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+1. Download, install, and setup the required tooling
+2. Download and install Tabula Rasa
+3. Patch Tabula Rasa using a custom patcher
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons.

File: docs/setup.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+# Setup Guide
+This guide will help you install and setup the required tools to run Tabula Rasa using Rasa.NET. There are a few steps:
+
+1. Download, install, and setup the required tooling
+2. Download and install Tabula Rasa
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons.

File: docs/setup.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+3. Patch Tabula Rasa using a custom patcher
+4. Download or clone the code
+5. Build and run the code from Visual Studio
+6. Launch Tabula Rasa
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons.

File: docs/setup.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+- For help, follow [the documentation](https://dev.mysql.com/doc/refman/5.7/en/mysql-installer-setup.html)
+- Set the `root` user password to something unique when prompted after installation
+
+### Install Git
+Git is a version control system and will allow you to download the code. Alternatively, if you're not planning on developing for Rasa.NET you can download a .zip of the code directly from GitHub.
+
+- Download [Git](https://git-scm.com/downloads)
+- Run the installer. If you're not sure of what options to select, choose the defaults
+
+## Download and install Tabula Rasa
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons. (In the whole section too.)

File: docs/setup.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+You'll need the Tabula Rasa game client. Below is the demo version which was made freely available. 
+
+- [Download and install the demo game client](http://infiniterasa.org/tools/tabularasa_demo.exe).
+- Create a shortcut to the game client on the desktop. You'll need to find the executable in the path where you installed the game. i.e. `C:\ProgramFiles (x86)\TabulaRasa\NCSoft\TabulaRasa\tabula_rasa.exe`
+  - Right-click on `tabular_rasa.exe` and choose `Send to > Desktop (create shortcut)`
+  - Go to the desktop and right-click on the newly created shortcut and choose `Properties`
+  - Select the `Shortcut` tab
+  - Edit the `Target` box to include `/NoPatch /AuthServer=localhost:2106` to the end of the file path. So the entire textbox should have `C:\ProgramFiles (x86)\TabulaRasa\NCSoft\TabulaRasa\tabula_rasa.exe /NoPatch /AuthServer=localhost:2106`
+
+## Patch Tabula Rasa using the custom patcher
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons. (In the whole section too.)

File: docs/setup.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+  - In the new tab, name the first schema `rasaauth` and click `Apply`
+  - Repeat this two more times for `rasachar` and `rasaworld`
+- Double-click `rasaauth` in the `SCHEMAS` list which makes it active
+- Go to `File > Open SQL Script`
+- Select `Rasa.NET\database\full\auth_database.sql` file in the code directory that you clone in the above steps
+- Execute the script to create the required tables by clicking the icon that looks like a lightning bolt
+  - Repeat this process for the `Rasa.NET\database\full\char_database.sql` and `Rasa.NET\database\full\world_database.sql` files - **but first double-click the matching schema on the left to make it the active selection**
+- Next, you'll have to execute each of the sql files in the `Rasa.NET\database\patches` folder
+  - Follow similar steps as above, making sure to make the correct schema active and execute each patch script against the matching schema in sequential order. 
+- Select the `rasaauth` schema and doubl-click it to make it active
```
Comment (Blumster) [inline docs/setup.md]: You can create account to your own liking in the authentication server command prompt with your username, email and password. Would be better if you have documented the process how to do that.

File: docs/setup.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+## Build and run the code from Visual Studio
+You should be ready to compile Rasa.NET and run the servers. 
+
+- Launch Visual Studio and open the `Rasa.NET.sln` file in the code repository
+- Build the solution
+- Run the `Rasa.Auth` project via `Debug > Start without Debugging`
+- Run the `Rasa.Game` project via `Debug > Start Debugging`
+
+## Launch Tabula Rasa
```
Comment (Blumster) [inline docs/setup.md]: Don't include copyrighted names for legal reasons.

**Review by Blumster — CHANGES_REQUESTED** [review]

**Comment by infiniterasadev** [comment]
Fixed!

File: docs/setup.md [hunk 10]
```diff
@@ -108,11 +102,16 @@ You should be ready to compile Rasa.NET and run the servers.
 - Run the `Rasa.Auth` project via `Debug > Start without Debugging`
 - Run the `Rasa.Game` project via `Debug > Start Debugging`
 
+### Create a game user
+The authentication server can be used to create a user. 
+
+- Run the command `create test@test.com test test` in the Authentication server terminal to create a new user
```
Comment (Blumster) [inline docs/setup.md]: Specify the usage and that they can create their own account, anything they want, and give a basic usage, like you did now.

**Comment by Dahrkael** [comment]
Regarding the screenshot, I'd enforce the same thing as @Blumster requested. Blur that part and blame whoever put that there.

**Review by Blumster — CHANGES_REQUESTED** [review]

**Review by Blumster — CHANGES_REQUESTED** [review]
Please modify the Download and install the game, remove any reference to an actual game. (Paths, exe names, ...), and refer to the forum about downloading the game. Only include the start of the game with specific parameters.

**Review by Blumster — APPROVED** [review]

**Pull request merged**